modules/basics/src/test/java/com/opengamma/basics/date/HolidayCalendarTest.java

```diff
@@ -483,6 +486,37 @@ public void test_previousOrSame_null() {
   }
 
   //-------------------------------------------------------------------------
+  @DataProvider(name = "nextLastOrSameInMonth")
+  static Object[][] data_nextLastOrSameInMonth() {
+    return new Object[][] {
+        {THU_2014_07_10, THU_2014_07_10},
```

Comment:
Just that if you add a comment against the data, the test is the one controlling the calendar so there is a slight disconnect